src/test/java/BaseTest.java

```diff
@@ -1,7 +1,18 @@
-import org.testng.annotations.BeforeSuite;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.*;
 
-public class BaseTest {
+import java.time.Duration;
 
+public class BaseTest {
+    protected WebDriver driver;
+    String url;
+    String email="holostenco.yuliya@gmail.com";
+    String password ="te$t$tudent";
```

Comment:
**Needless Repetition!**  You can delete these variables because they are in the class
`class LoginPage - login()` )